src/views/Swap/Swap.vue

```diff
@@ -65,7 +65,7 @@
           :asset="selectedQuote.bridgeAsset"
         />
       </InfoNotification>
-      <InfoNotification v-else-if="cannotCoverMinimum && !insufficientFundsError">
+      <InfoNotification v-else-if="cannotCoverMinimum || insufficientFundsError">
```

Comment:
Not sure if the condition should be changed like this. 

Current logic was  - if you  Cannot cover minimum && there is no Inssuficient Funds error - then show some message 

Now it is  - if you cannot cover minimum or   Inssuficient Funds error - it means that if the funds are inssuficient it will show an error similar to the error when use r cannot cover minimum.  With our new implementation when insufficientFundsError is triggered - the review button in the bottom left stays disabled in status insufficient funds and it does not  result in showing extra error message

Reply:
Yeah, the requirements from the linear task are exactly this, expecting to display error if there are insufficient funds::

https://linear.app/liquality/issue/LIQ-1492/in-case-user-balance-is-less-than-min-balance-during-swap-then-correct